scripts/bash/run_enceladus.sh

```diff
@@ -100,6 +107,14 @@ case $key in
     EXECUTOR_MEMORY="$2"
     shift 2 # past argument and value
     ;;
+  --dra-executor-cores)
+    DRA_EXECUTOR_CORES="$2"
+    shift 2 # past argument and value
+    ;;
+  --dra-executor-memory)
+    DRA_EXECUTOR_MEMORY="$2"
+    shift 2 # past argument and value
+    ;;
```

Comment:
**Major:** There is no way how to disable DRA other then changing the value in `enceladus_env.sh`. Didn't we want to have a switch `--disable-dra`?

Reply:
You can use `--enable-dra false`

Reply:
Was not aware of this switch already being there. 🙈